### DISCUSSION TITLE
How to? API create/update attributes terms, Swatch Label and Swatch Color hex

### DISCUSSION THREAD

**Discussion by shaiazran** [woocommerce/woocommerce#39765] [General]
I am using the Woocommerce API to create products for my website. 
 How should I send the colour hex code to the attribute term and turn on “Enable swatch”.

I want to update this: 
Swatch Label -  for example, 'Pink2'
Swatch Color hex #FE9684
 
I'm using the API (python)
In the API doc.  I don't find the Parameter for it too

{
  "id": 23,
  "name": "XXS",
  "slug": "xxs",
  "description": "",
  "menu_order": 1,
  "count": 1,
  "_links":  .....
 
print(wcapi.post("products/attributes/2/terms", data).json())

**Comment by barryhughes** [comment]
I don't generally work with the Python library, but thinking about things from a more general perspective it seems like you are missing the attribute term ID. That is, instead of:

`products/attributes/<ATTR_ID>/terms`

You may want:

`products/attributes/<ATTR_ID>/terms/<TERM_ID>`

Ref: [woocommerce.github.io/woocommerce-rest-api-docs/#update-an-attribute-term](https://woocommerce.github.io/woocommerce-rest-api-docs/#update-an-attribute-term)
